### ISSUE TITLE
Fixed vertical position for navText?

### ISSUE THREAD

**Issue by Jay-PC** [vitalii-andriiovskyi/ngx-owl-carousel-o#150]
Currently, the navText chevrons < > appear to be vertically place dynamically, based on frame height--they appear in to be set to the middle. However, we have a need for frames to have varying heights and would like to set the navText to a fixed location based from the top so that the user doesn't have to look for them on every click. Is that possible? Thank you.

**Comment by vitalii-andriiovskyi** [comment]
@Jay-PC 

Yes it's possible through css. Usually `position: absolute` and `left`, `top`... help.
